cypress/fixtures/items.js

```diff
@@ -123,6 +124,56 @@ export const FOLDER_WITH_PINNED_ITEMS = {
   ],
 };
 
+export const FOLDER_WITH_HIDDEN_ITEMS = {
+  items: [
+    {
+      ...DEFAULT_FOLDER_ITEM,
+      id: 'ecafbd2a-5688-11eb-ae93-0242ac130008',
+      name: 'parent folder',
+      path: 'ecafbd2a_5688_11eb_ae93_0242ac130008',
+      extra: {
+        image: 'someimageurl',
+      },
+      settings: {
+        isPinned: false,
+        showChatbox: false,
+      },
+    },
+    {
+      ...DEFAULT_FOLDER_ITEM,
+      id: 'fdf09f5a-5688-11eb-ae93-0242ac130009',
+      name: 'Visible',
+      path: 'ecafbd2a_5688_11eb_ae93_0242ac130008.fdf09f5a_5688_11eb_ae93_0242ac130009',
+      extra: {
+        image: 'someimageurl',
+      },
+      settings: {
+        isPinned: false,
+        showChatbox: false,
+      },
+    },
+    {
+      ...DEFAULT_FOLDER_ITEM,
+      id: 'fdf09f5a-5688-11eb-ae93-0242ac130010',
+      name: 'Hidden',
+      path: 'ecafbd2a_5688_11eb_ae93_0242ac130008.fdf09f5a_5688_11eb_ae93_0242ac130010',
+      extra: {
+        image: 'someimageurl',
+      },
+      settings: {
+        isPinned: false,
+        showChatbox: false,
+      },
+      tags: [
+        {
+          tagId: false,
+        }
+      ],
```

Comment:
I might be wrong, but it doesn't look hidden? It's hidden with the taf right?

Reply:
By default the value of REACT_APP_HIDDEN_ITEM_TAG_ID is false. So it's necessary to edit the CI env vars to add a value other than false